spec/features/edit_document_associations_spec.rb

```diff
@@ -18,11 +21,16 @@
   end
 
   def given_there_is_a_document
-    association_schema = build(:association_schema, type: "multi_association", id: "association_id")
-    document_type_schema = build(:document_type_schema, associations: [association_schema])
+    multi_association_schema = build(:association_schema, type: "multi_association", id: "multi_association_id")
+    single_association_schema = build(:association_schema, type: "single_association", id: "single_association_id")
+    document_type_schema = build(:document_type_schema, associations: [multi_association_schema, single_association_schema])
     multi_association_linkables = [initial_association, association_to_select_1, association_to_select_2]
-    publishing_api_has_linkables(multi_association_linkables, document_type: association_schema["document_type"])
-    initial_associations = { association_schema["id"] => [initial_association["content_id"]] }
+    publishing_api_has_linkables(multi_association_linkables, document_type: multi_association_schema["document_type"])
+    publishing_api_has_linkables([selected_association, other_association], document_type: single_association_schema["document_type"])
```

Comment:
Could we reuse the mock associations we already had? It would reduce the number of variables to track, is what I’m thinking.